src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -327,6 +327,66 @@ export default defineConfig({
 });
 ```
 
+### Dev features
+
+When running `astro dev`, the adapter enables several Netlify platform features to ensure the environment matches production as closely as possible. These include:
+
+- A local [Netlify Image CDN](https://docs.netlify.com/build/image-cdn/overview/#local-development) server. This is used for [images](#netlify-image-cdn-support) by default
+- A local [Netlify Blobs](https://docs.netlify.com/build/data-and-storage/netlify-blobs/) server. This is used for [sessions](#sessions) by default
+- [Redirects, rewrites](https://docs.netlify.com/routing/redirects/) and [headers](https://docs.netlify.com/routing/headers/#app) from your Netlify config
```

Comment:
the links changed a bit (they redirect properly, but it doesn't hurt to update). Also, the `#app` header for the headers link doesn't exist anymore, so I omitted it.
```suggestion
- [Redirects, rewrites](https://docs.netlify.com/manage/routing/redirects/overview/) and [headers](https://docs.netlify.com/manage/routing/headers/) from your Netlify config
```